ui-v2/app/services/repository/type/event-source.js

```diff
@@ -6,8 +6,27 @@ import LazyProxyService from 'consul-ui/services/lazy-proxy';
 import { cache as createCache, BlockingEventSource } from 'consul-ui/utils/dom/event-source';
 
 const createProxy = function(repo, find, settings, cache, serialize = JSON.stringify) {
-  // proxied find*..(id, dc)
   const client = get(this, 'client');
+  const store = get(this, 'store');
+  // custom createEvent, here used to reconcile the ember-data store for each tick
+  const createEvent = function(result, configuration) {
+    const event = {
+      type: 'message',
+      data: result,
+    };
+    const meta = get(event.data || {}, 'meta');
+    if (typeof meta.date !== 'undefined') {
+      // unload anything older than our current sync date/time
+      store.peekAll(repo.getModelName()).forEach(function(item) {
+        const date = get(item, 'SyncTime');
+        if (typeof date !== 'undefined' && date != meta.date) {
```

Comment:
The comment on line 19 suggests an inequality here. I'm assuming the `!=` is functionally equivalent to a `<` since there will only ever be two dates in the store (the old one and the new one)?